Automattic/jetpack
# Jetpack Boost: Refactor Image Size Analysis

## Proposed changes:
Refactor the entire image guide UI to React.js

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
Check out the branch, build all the things, play use the image size analysis module and compare to the current trunk _(don't compare to svelte version because transitions don't exist in our React app at the moment - we don't have a way to use transitions right now)_
